src/Filter/CheckboxFilter/CheckboxFilter.test.tsx

```diff
@@ -228,11 +228,7 @@ describe("CheckboxFilter", () => {
         "A",
         "b",
         "B",
-        "C",
-        "Gate 1",
-        "Gate 4",
-        "Gate 5",
-        "Gate 10"
+        "C"
```

Comment:
Removed the last 4 options ("Gate 1", "Gate 4", "Gate 5", "Gate 10") from the expected array because VirtualizedAutocomplete renders only the first 10 options initially for performance optimization.